osu.Game/Overlays/MusicController.cs

```diff
@@ -53,6 +57,39 @@ public class MusicController : FocusedOverlayContainer
         private Container dragContainer;
         private Container playerContainer;
 
+        private bool showPlaylistOnceAvailable;
+
+        private bool allowBeatmapChange = true;
+
+        public bool AllowBeatmapChange

```

Comment:
This should probably be a bindable, and use e.g. `prevButton.BindTo(AllowBeatmapChange)`.